erpnext/hr/doctype/employee_referral/employee_referral.json

```diff
@@ -0,0 +1,274 @@
+{
+ "actions": [],
+ "autoname": "format:HR-REF-{####}",
+ "creation": "2021-03-23 14:54:45.047051",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "first_name",
+  "last_name",
+  "full_name",
+  "email",
+  "contact_no",
+  "resume",
+  "column_break_6",
+  "date",
+  "status",
+  "current_employer",
+  "current_job_title",
+  "for_designation",
+  "referrer_details_section",
+  "employee",
+  "employee_name",
+  "column_break_14",
+  "is_applicable_for_referral_compensation",
+  "department",
+  "additional_information_section",
+  "why_this_candidate_is_qualified_for_this_position",
+  "column_break_17",
+  "work_references",
+  "amended_from"
+ ],
+ "fields": [
+  {
+   "fieldname": "first_name",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "First Name ",
+   "reqd": 1
+  },
+  {
+   "fieldname": "last_name",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Last Name",
+   "reqd": 1
+  },
+  {
+   "fieldname": "full_name",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Full Name",
+   "read_only": 1
+  },
+  {
+   "fieldname": "contact_no",
+   "fieldtype": "Data",
+   "label": "Contact No.",
+   "options": "Phone"
+  },
+  {
+   "fieldname": "current_employer",
+   "fieldtype": "Data",
+   "label": "Current Employer "
+  },
+  {
+   "fieldname": "column_break_6",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "date",
+   "fieldtype": "Date",
+   "label": "Date",
+   "reqd": 1
+  },
+  {
+   "allow_on_submit": 1,
+   "fieldname": "status",
+   "fieldtype": "Select",
+   "label": "Status",
+   "no_copy": 1,
+   "options": "Pending\nIn Process\nAccepted\nRejected",
+   "permlevel": 1,
+   "read_only": 1,
+   "reqd": 1
+  },
+  {
+   "fieldname": "current_job_title",
+   "fieldtype": "Data",
+   "label": "Current Job Title"
+  },
+  {
+   "fieldname": "resume",
+   "fieldtype": "Attach",
+   "label": "Resume"
+  },
+  {
+   "fieldname": "referrer_details_section",
+   "fieldtype": "Section Break",
+   "label": "Referrer Details"
+  },
+  {
+   "fieldname": "employee",
+   "fieldtype": "Link",
+   "label": "Employee ",
+   "options": "Employee",
+   "reqd": 1
```

Comment:
```suggestion
   "fieldname": "referred_by",
   "fieldtype": "Link",
   "label": "Referred By ",
   "options": "Employee",
   "reqd": 1
```
I think we should label this field as Referred By instead of Employee.

Reply:
Maybe we can rename it to Referrer and Referrer Name.. but think an employee is pretty standard. As there is already a separate section for Referral(Refer to screenshot). and if we name it Referred By... what should be a label for **Employee name**...
![image](https://user-images.githubusercontent.com/32095923/116040912-392ed400-a68a-11eb-9fdd-0646bd9b5faf.png)


Maybe we can rename it to Referrer and Referrer name.. but think an employee is pretty standard.

Reply:
Yes, let's rename it to Referrer and Referrer Name

Reply:
Done